code/modules/tgui/tgui_window.dm

```diff
@@ -209,7 +209,13 @@
 /datum/tgui_window/proc/send_message(type, payload, force)
 	if(!client)
 		return
+
+	var/tick_usage = TICK_USAGE
 	var/message = TGUI_CREATE_MESSAGE(type, payload)
+	var/delta = TICK_USAGE - tick_usage
+
+	SStgui.queued_calculate_deltas += list(list(message, TICK_USAGE_TO_MS(delta)))
```

Comment:
```suggestion
	SStgui.queued_calculate_deltas += list(list(message, TICK_DELTA_TO_MS(delta)))
```
TICK_USAGE_TO_MS is for when you want to pass the old tick_usage and use the current tick_usage to generate a delta

Reply:
whoops, i had this open too